frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -548,3 +554,23 @@ export function useClusterNodesColumn(): IAcmTableColumn<Cluster> {
         },
     }
 }
+
+export function useClusterCreatedDateColumn(clusterDeployments: ClusterDeployment[]): IAcmTableColumn<Cluster> {
```

Comment:
Would it be simpler to add the desired `creationTimeStamp` property to the `Cluster` type which is generated from various resources in `get-cluster.ts` module? We could avoid having to reach for `ClusterDeployments` in this component and most likely the sorting would be as simple as `sort: 'creationTimeStamp'`.

What do you think?

Reply:
Yes, it works better in this way. I've changed it